README.rst

```diff
@@ -43,6 +53,17 @@ How can I run the tests?
    will reduce the danger of race conditions when building containers)
 4. Run ``tox -e $language_stack -- -n auto``
 
+Environment variables
+^^^^^^^^^^^^^^^^^^^^^
+
+One can use the following environment variables to configure the behaviour of the BCI-Tests
+
+.. code-block:: shell-session
+
+    # export OS_VERSION=15.6                         # Target SLES version
+    # export CONTAINER_RUNTIME=podman                # Defaults to podman
+    # export TARGET=ibs-cr                           # ibs-cr = ToTest, ibs = Released, obs = Tumbleweed
```

Comment:
```suggestion
    $ export OS_VERSION=15.6                         # Target SLES version
    $ export CONTAINER_RUNTIME=podman                # Defaults to podman
    $ export TARGET=ibs-cr                           # ibs-cr = ToTest, ibs = SUSE:SLE-15-SP*:Update:BCI, obs = devel:BCI:*
```

Reply:
Thanks for the correction. I now also added `factory-totest`